src/cross_module_fn.c

```diff
@@ -293,6 +293,14 @@ preprocess_query_tsl_default_fn_community(Query *parse, int *cursor_opts)
 	/* No op in community licensed code */
 }
 
+static PGFunction
+bloom1_get_hash_function_default(Oid type, FmgrInfo **finfo)
+{
+	ts_license_enable_module_loading();
```

Comment:
if you are loading the license in the same function you will still be in this functions context and only for subsequent calls the function pointer would be updated